### ISSUE TITLE
Adapt Accessibility tests: co-12 no individual question scoring for visually impaired users

### ISSUE THREAD

**Issue by shauncg** [adaptlearning/adapt_framework#422]
Because no feedback pop-ups are used in the assessment JAWS doesn't have any way of interpreting the correct/incorrect marking, so visually impaired users will have no way of knowing if they were correct or not until they reach the end of the assessment and get their overall score and feedback.
